### PR TITLE
Event debug fix

### PR DESCRIPTION
**Pull request by RadekVana**
BFW-1287

### CODE REVIEW

File: src/gui/screen_printing.hpp [hunk 1]
```diff
@@ -43,7 +43,7 @@ constexpr static const size_t POPUP_MSG_DUR_MS = 5000;
 constexpr static const size_t MAX_END_TIMESTAMP_SIZE = 14 + 12 + 5; // "dd.mm.yyyy at hh:mm:ss" + safty measures for 3digit where 2 digits should be
 constexpr static const size_t MAX_TIMEDUR_STR_SIZE = 9;
 
-class screen_printing_data_t : public IScreenPrinting {
+class screen_printing_data_t : public AddSuperWindow<IScreenPrinting> {
     static constexpr const char *caption = "PRINTING";
```
Comment (DRracer) [inline src/gui/screen_printing.hpp:47]: N_("PRINTING") ?

File: src/gui/screen_printing_serial.hpp [hunk 2]
```diff
@@ -9,7 +9,7 @@
 
 static constexpr btn_resource res_disconnect = { IDR_PNG_menu_icon_disconnect, N_("Disconnect") };
 
-class screen_printing_serial_data_t : public IScreenPrinting {
+class screen_printing_serial_data_t : public AddSuperWindow<IScreenPrinting> {
     static constexpr const char *caption = "SERIAL PRT.";
```
Comment (DRracer) [inline src/gui/screen_printing_serial.hpp:13]: Also not translated? Is it correct?

File: src/gui/screen_printing.hpp (hunk 1 above)
Comment (RadekVana) [reply src/gui/screen_printing.hpp]: fixed

File: src/gui/screen_printing_serial.hpp (hunk 2 above)
Comment (RadekVana) [reply src/gui/screen_printing_serial.hpp]: fixed

**Review by DRracer — APPROVED** [review]

**Pull request merged**